spec/models/profile_spec.rb

```diff
@@ -431,53 +713,101 @@
     it 'does not activate a profile if it encounters a transaction error' do
       profile = create(
         :profile,
+        :password_reset,
+        :verified,
         user: user,
-        active: false,
-        deactivation_reason: :password_reset,
-        verified_at: 1.day.ago,
       )
 
       allow(profile).to receive(:update!).and_raise(RuntimeError)
 
+      expect(profile.activated_at).to be_present
+      expect(profile.active).to eq(false)
+      expect(profile.deactivation_reason).to eq('password_reset')
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at).to be_nil
+      expect(profile.initiating_service_provider).to be_nil
+      expect(profile.verified_at).to be_present
+
       suppress(RuntimeError) do
         profile.activate_after_password_reset
       end
 
+      expect(profile.activated_at).to be_present
+      expect(profile.active).to eq(false)
+      expect(profile.deactivation_reason).to eq('password_reset')
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at).to be_nil
+      expect(profile.initiating_service_provider).to be_nil
+      expect(profile.verified_at).to be_present
+
       expect(profile.deactivation_reason).to eq('password_reset')
       expect(profile).to_not be_active
     end
   end
 
   describe '#activate_after_passing_in_person' do
     it 'activates a profile if it passes in person proofing' do
-      profile = user.profiles.create
-      profile.active = false
-      profile.fraud_review_pending_at = 1.day.ago
-      profile.deactivation_reason = :in_person_verification_pending
+      profile = create(
+        :profile,
+        :fraud_review_pending,
+        :in_person_verification_pending,
```

Comment:
This is an invalid combination of characteristics. Fraud review pending doesn't apply to in person proofing.